kratos/python_scripts/modelers/import_model_part_modeler.py

```diff
@@ -0,0 +1,50 @@
+import KratosMultiphysics
+
+class ImportModelPartModeler(KratosMultiphysics.Modeler):
```

Comment:
Please change it to ImportMDPAModeler to have it by the type.

Reply:
Done.